scripts/sdk/build.sh

```diff
@@ -11,7 +11,7 @@ cp doc/SDK.md target/README.md
 pushd target
 pushd asset-build/assets
 mv main.*.js ../../main.js
-mv index.*.css ../../style.css
+mv *light.*.css ../../style.css
```

Comment:
I'd suggest creating a copy of the light css file for backwards compatibility, but also rename the hashed file to not have a content hash (so imports don't need to include the content hash).
can we rename the dark theme file to not have a content hash as well?